package/verification-template/build.gradle

```diff
@@ -55,6 +55,12 @@ dependencies {
 	thirdParty("com.diffplug.spotless:spotless-plugin-gradle:5.2.0") {
 		because "ring-of-recoil-notifier"
 	}
+	thirdParty("com.google.apis:google-api-services-sheets:v4-rev581-1.25.0") {
+		because "elysiumevents-plugin"
+	}
+	thirdParty("com.google.guava:guava:20.0") {
```

Comment:
I don't think it will be possible to override the guava version at runtime, since we control it as it is a direct dep of runelite-client. You should probably just exclude the dep from whatever is pulling it in.